### PR TITLE
Templates updated

### PR DESCRIPTION
**Pull request by mmexvr**
**Describe pull-request**</br>
Describe what the pull-request is about
- Pull-request template
- issues template

**Additional context**<br/>
Add any other context about the pull-request here.
Matches Corporate-ui templates

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -1,52 +1,52 @@
 ---
 name: Bug report
-about: Use this option when you found error on scania-theme
-title: Bug - title
-labels: bug
+about: Use this option when you've found an error caused by Corporate UI.
```
Comment (helloimela) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Corporate UI or scania-theme ?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -1,52 +1,52 @@
...
+_A clear and concise description of what you expected to happen._
 
-**Screenshots**
-If applicable, add screenshots to help explain your problem.
+**Screenshots**  
+_If applicable, add screenshots to help explain your problem._
 
-**Version of Corporate-ui**
-E.g 4.0.0-alpha.1
+**Version of Corporate-ui**
```
Comment (helloimela) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Version of Scania-theme

Comment (helloimela) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Maybe both corp ui and theme ?

Comment (mmexvr) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Changed to scania theme for now

**Review by helloimela — APPROVED** [review]

**Pull request merged**